src/v2/Components/ArtworkFilter/ArtworkFilters/__tests__/SizeFilter2.rtl.jest.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { ReactElement } from "react"
+import {
+  render as originalRender,
+  RenderOptions,
+  screen,
+} from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+
+import {
+  ArtworkFilterContextProps,
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "v2/Components/ArtworkFilter/ArtworkFilterContext"
+import { SizeFilter2 } from "../SizeFilter2"
+
+const render = (ui: ReactElement, options: RenderOptions = {}) =>
+  originalRender(ui, { wrapper: Wrapper, ...options })
+
+const Wrapper: React.FC = ({ children }) => {
+  return (
+    <ArtworkFilterContextProvider>
+      <ClearAllButton />
+      {children}
+      <ArtworkFilterContextInspector />
+    </ArtworkFilterContextProvider>
+  )
+}
```

Comment:
This pattern of overriding render with any custom setup/wrappers/providers is [recommended](https://testing-library.com/docs/react-testing-library/setup#custom-render).